### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by nahcbr** [turkaysoft/glow#5] [Q&A] [answered]
Hi,
How can I translate Glow?
Thanks.
Nelson Carvalho

**Comment by turkaysoft** [comment]
> Hi, How can I translate Glow? Thanks. Nelson Carvalho

Hello, in which language do you want to translate?

**Reply by nahcbr** [reply, marked as answer]
Portuguese (Brazilian) (Português Brasil).

**Reply by turkaysoft** [reply]
Thank you very much. 

I understand that you want to help, but I don't plan to add any languages other than English and Turkish for Glow for a long time.

This is because English is the main language globally.
Of course, I may change my mind. 

If I change my mind, I can share it as an announcement.

Best regards.

**Comment by turkaysoft** [comment]
@nahcbr 

Hello again

If you are available, can you translate Glow v24.08 into Portuguese or Brazilian Portuguese?

I would like to add the relevant languages.

Best regards.

**Comment by nahcbr** [comment]
Hi,
Yes.
I would like to translate.
Best regards.
